mongo/bounties/final.json

```diff
@@ -0,0 +1,85 @@
+[
+  {
+    "_id": "60f6585b453e70eed340e8e3",
+    "season": 1,
+    "title": "Make new website landing page",
+    "description": "Description for new website landing page with specs",
+    "criteria": "A completed bounty has the following deliverables...",
+    "reward": {
+      "currency": "BANK",
+      "amount": 10000.00
+    },
+    "applicableGuilds": [
+      {
+        "guildName": "Developers Guild",
+        "id": "60f6585b453e70eed340e8e3"
```

Comment:
What is the purpose of having an ID associated with the guildName? Since users won't be entering the ID when they create the bounty, the name of a guild will have to be unique to the user. I think controlling this input to the list of correctly spelled guild names (perhaps as an Enum for DEGEN?) will be helpful to the user experience

Reply:
Good question, basically this is the ID that is assigned for every discord role or in this case the guild name. 

<img width="937" alt="Screen Shot 2021-08-01 at 12 41 28 PM" src="https://user-images.githubusercontent.com/6130035/127778744-0306bb1c-9284-429c-b0ea-e90e856358f7.png">

The idea here is that for every guild name that the bounty might be applicable for there is an associated ID in discord that would be matching and lookups easier. For now the field is not required so I expect it to be null and we'll probably refine it in later iterations